BeeSwift/GalleryViewController.swift

```diff
@@ -373,6 +384,21 @@ class GalleryViewController: UIViewController, UICollectionViewDelegateFlowLayou
     }
     
     @objc func didFetchGoals() {
+        let thumbs = self.goals.map{ $0.cacheBustingThumbUrl }
+        let graphs = self.goals.map{ $0.cacheBustingGraphUrl }
```

Comment:
We could certainly debate about the right number here or if this is needed. We download/cache all thumbnails and all graphs (thumbnails first). 
What about slow connections? What about accounts with 70+ goals? Is beeminder server-side checking logs of graphs obtained for analytics to indicate a user navigated to a particular place in the app? 
I guess if we have any doubts, we could leave the graphs downloaded/cached when the user views the goal (GoalVC).